diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/LICENSE b/node_modules/@homebridge/node-pty-prebuilt-multiarch/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/README.md b/node_modules/@homebridge/node-pty-prebuilt-multiarch/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/binding.gyp b/node_modules/@homebridge/node-pty-prebuilt-multiarch/binding.gyp
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/.editorconfig b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/.editorconfig
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/.drone.yml b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/.drone.yml
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/.gitattributes b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/.gitattributes
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/LICENSE b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/Makefile b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/Makefile
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/README.md b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/RELEASES.md b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/RELEASES.md
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/VERSION.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/VERSION.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/configure b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/configure
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/BufferResizeTests.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/BufferResizeTests.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ChangeScreenBuffer.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ChangeScreenBuffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ClearConsole.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ClearConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConinMode.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConinMode.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConinMode.ps1 b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConinMode.ps1
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConoutMode.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ConoutMode.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DebugClient.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DebugClient.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DebugServer.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DebugServer.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DumpLines.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/DumpLines.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/EnableExtendedFlags.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/EnableExtendedFlags.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP437-Consolas.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP437-Consolas.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP437-Lucida.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP437-Lucida.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP932.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP932.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP936.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP936.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP949.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP949.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP950.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/CP950.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/MinimumWindowWidths.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/MinimumWindowWidths.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/Results.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/Results.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/Windows10SetFontBugginess.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Font-Report-June2016/Windows10SetFontBugginess.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FontSurvey.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FontSurvey.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FormatChar.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FormatChar.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FreezePerfTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/FreezePerfTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetCh.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetCh.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetConsolePos.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetConsolePos.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetFont.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/GetFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/IdentifyConsoleWindow.ps1 b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/IdentifyConsoleWindow.ps1
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/IsNewConsole.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/IsNewConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/MouseInputNotes.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/MouseInputNotes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/MoveConsoleWindow.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/MoveConsoleWindow.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Notes.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Notes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/OSVersion.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/OSVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferFreezeInactive.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferFreezeInactive.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferTest2.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ScreenBufferTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SelectAllTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SelectAllTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetBufferSize.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetBufferSize.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetCursorPos.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetCursorPos.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetFont.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetWindowRect.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/SetWindowRect.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ShowArgv.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ShowArgv.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ShowConsoleInput.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/ShowConsoleInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Spew.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Spew.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/TestUtil.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/TestUtil.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeDoubleWidthTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeDoubleWidthTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeWideTest1.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeWideTest1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeWideTest2.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnicodeWideTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnixEcho.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/UnixEcho.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Utf16Echo.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Utf16Echo.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/VeryLargeRead.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/VeryLargeRead.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/VkEscapeTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/VkEscapeTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10ResizeWhileFrozen.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10ResizeWhileFrozen.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10WrapTest1.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10WrapTest1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10WrapTest2.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win10WrapTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Echo1.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Echo1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Echo2.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Echo2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test1.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test2.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test3.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Test3.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Write1.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/Win32Write1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/WindowsBugCrashReader.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/WindowsBugCrashReader.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/WriteConsole.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/WriteConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/build32.sh b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/build32.sh
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/build64.sh b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/build64.sh
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/color-test.sh b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/color-test.sh
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/font-notes.txt b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/font-notes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/winbug-15048.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/misc/winbug-15048.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/build-pty4j-libpty.bat b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/build-pty4j-libpty.bat
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/common_ship.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/common_ship.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/make_msvc_package.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/make_msvc_package.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/ship.py b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/ship/ship.py
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Agent.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Agent.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Agent.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Agent.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/AgentCreateDesktop.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/AgentCreateDesktop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/AgentCreateDesktop.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/AgentCreateDesktop.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleFont.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleFont.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleFont.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInput.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInput.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInput.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInputReencoding.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInputReencoding.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInputReencoding.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleInputReencoding.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleLine.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleLine.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleLine.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/ConsoleLine.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Coord.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Coord.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DebugShowInput.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DebugShowInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DebugShowInput.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DebugShowInput.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DefaultInputMap.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DefaultInputMap.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DefaultInputMap.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DefaultInputMap.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DsrSender.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/DsrSender.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/EventLoop.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/EventLoop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/EventLoop.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/EventLoop.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/InputMap.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/InputMap.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/InputMap.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/InputMap.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/LargeConsoleRead.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/LargeConsoleRead.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/LargeConsoleRead.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/LargeConsoleRead.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/NamedPipe.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/NamedPipe.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/NamedPipe.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/NamedPipe.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Scraper.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Scraper.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Scraper.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Scraper.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/SimplePool.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/SimplePool.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/SmallRect.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/SmallRect.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Terminal.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Terminal.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Terminal.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Terminal.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/UnicodeEncoding.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/UnicodeEncoding.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/UnicodeEncodingTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/UnicodeEncodingTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32Console.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32Console.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32Console.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32Console.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32ConsoleBuffer.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32ConsoleBuffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32ConsoleBuffer.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/Win32ConsoleBuffer.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/main.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/main.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/agent/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/configurations.gypi b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/configurations.gypi
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/debugserver/DebugServer.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/debugserver/DebugServer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/debugserver/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/debugserver/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/include/winpty.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/include/winpty.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/include/winpty_constants.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/include/winpty_constants.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/AgentLocation.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/AgentLocation.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/AgentLocation.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/AgentLocation.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/LibWinptyException.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/LibWinptyException.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/WinptyInternal.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/WinptyInternal.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/winpty.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/libwinpty/winpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/AgentMsg.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/AgentMsg.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/BackgroundDesktop.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/BackgroundDesktop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/BackgroundDesktop.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/BackgroundDesktop.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Buffer.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Buffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Buffer.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Buffer.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/DebugClient.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/DebugClient.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/DebugClient.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/DebugClient.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GenRandom.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GenRandom.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GenRandom.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GenRandom.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GetCommitHash.bat b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/GetCommitHash.bat
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Mutex.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/Mutex.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OsModule.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OsModule.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OwnedHandle.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OwnedHandle.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OwnedHandle.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/OwnedHandle.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/PrecompiledHeader.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/PrecompiledHeader.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringBuilder.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringBuilder.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringBuilderTest.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringBuilderTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringUtil.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringUtil.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringUtil.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/StringUtil.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/TimeMeasurement.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/TimeMeasurement.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/UnixCtrlChars.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/UnixCtrlChars.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/UpdateGenVersion.bat b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/UpdateGenVersion.bat
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsSecurity.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsSecurity.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsSecurity.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsSecurity.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsVersion.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsVersion.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WindowsVersion.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyAssert.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyAssert.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyAssert.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyAssert.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyException.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyException.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyException.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyException.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyVersion.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyVersion.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/WinptyVersion.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/winpty_snprintf.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/shared/winpty_snprintf.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/tests/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/tests/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/tests/trivial_test.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/tests/trivial_test.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/InputHandler.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/InputHandler.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/InputHandler.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/InputHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/OutputHandler.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/OutputHandler.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/OutputHandler.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/OutputHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/Util.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/Util.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/Util.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/Util.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/WakeupFd.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/WakeupFd.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/WakeupFd.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/WakeupFd.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/main.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/main.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/subdir.mk b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/unix-adapter/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/winpty.gyp b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/src/winpty.gyp
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/vcbuild.bat b/node_modules/@homebridge/node-pty-prebuilt-multiarch/deps/winpty/vcbuild.bat
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/conpty_console_list_agent.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/conpty_console_list_agent.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/conpty_console_list_agent.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/conpty_console_list_agent.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/eventEmitter2.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/index.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/index.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/interfaces.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/interfaces.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/interfaces.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/interfaces.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-file-path.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-file-path.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-file-path.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-file-path.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-loader.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-loader.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-loader.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/prebuild-loader.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/shared/conout.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/shared/conout.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/shared/conout.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/shared/conout.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/terminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/testUtils.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/testUtils.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/testUtils.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/testUtils.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/types.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/types.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/unixTerminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/utils.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/utils.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsConoutConnection.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsConoutConnection.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsConoutConnection.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsConoutConnection.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsPtyAgent.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.test.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.test.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/windowsTerminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/worker/conoutSocketWorker.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/worker/conoutSocketWorker.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/worker/conoutSocketWorker.js.map b/node_modules/@homebridge/node-pty-prebuilt-multiarch/lib/worker/conoutSocketWorker.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/check-prebuild.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/check-prebuild.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/increment-version.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/increment-version.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/install.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/install.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/post-install.js b/node_modules/@homebridge/node-pty-prebuilt-multiarch/scripts/post-install.js
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/conpty_console_list_agent.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/conpty_console_list_agent.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/eventEmitter2.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/eventEmitter2.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/eventEmitter2.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/eventEmitter2.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/index.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/interfaces.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/interfaces.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/native.d.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/native.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/prebuild-file-path.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/prebuild-file-path.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/prebuild-loader.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/prebuild-loader.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/shared/conout.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/shared/conout.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/terminal.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/terminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/terminal.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/terminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/testUtils.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/testUtils.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/tsconfig.json b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/tsconfig.json
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/types.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/pty.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/pty.cc
old mode 100644
new mode 100755
index 2c99536..87e3855
--- a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/pty.cc
+++ b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/pty.cc
@@ -41,7 +41,9 @@
 #include <util.h>
 #elif defined(__FreeBSD__)
 #include <libutil.h>
+#elif defined(__sun) || defined(__SunOS)
 #include <termios.h>
+#include <sys/stropts.h>
 #endif
 
 /* Some platforms name VWERASE and VDISCARD differently */
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/spawn-helper.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unix/spawn-helper.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unixTerminal.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unixTerminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unixTerminal.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/unixTerminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/utils.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/utils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty_console_list.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/conpty_console_list.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/path_util.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/path_util.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/path_util.h b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/path_util.h
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/winpty.cc b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/win/winpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsConoutConnection.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsConoutConnection.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsPtyAgent.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsPtyAgent.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsPtyAgent.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsPtyAgent.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsTerminal.test.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsTerminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsTerminal.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/windowsTerminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/worker/conoutSocketWorker.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/src/worker/conoutSocketWorker.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-arm64/OpenConsole.exe b/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-arm64/OpenConsole.exe
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-arm64/conpty.dll b/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-arm64/conpty.dll
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-x64/OpenConsole.exe b/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-x64/OpenConsole.exe
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-x64/conpty.dll b/node_modules/@homebridge/node-pty-prebuilt-multiarch/third_party/conpty/1.22.250204002/win10-x64/conpty.dll
old mode 100644
new mode 100755
diff --git a/node_modules/@homebridge/node-pty-prebuilt-multiarch/typings/node-pty.d.ts b/node_modules/@homebridge/node-pty-prebuilt-multiarch/typings/node-pty.d.ts
old mode 100644
new mode 100755
