diff --git a/node_modules/node-pty/LICENSE b/node_modules/node-pty/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/README.md b/node_modules/node-pty/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/binding.gyp b/node_modules/node-pty/binding.gyp
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/build/Makefile b/node_modules/node-pty/build/Makefile
old mode 100644
new mode 100755
index 77ec600..0ccd54c
--- a/node_modules/node-pty/build/Makefile
+++ b/node_modules/node-pty/build/Makefile
@@ -47,7 +47,6 @@ CXXFLAGS.target ?= $(CPPFLAGS) $(CXXFLAGS)
 LINK.target ?= $(LINK)
 LDFLAGS.target ?= $(LDFLAGS)
 AR.target ?= $(AR)
-PLI.target ?= pli
 
 # C++ apps need to be linked with g++.
 LINK ?= $(CXX.target)
@@ -61,7 +60,6 @@ CXXFLAGS.host ?= $(CPPFLAGS_host) $(CXXFLAGS_host)
 LINK.host ?= $(CXX.host)
 LDFLAGS.host ?= $(LDFLAGS_host)
 AR.host ?= ar
-PLI.host ?= pli
 
 # Define a dir function that can handle spaces.
 # http://www.gnu.org/software/make/manual/make.html#Syntax-of-Functions
@@ -135,9 +133,6 @@ quiet_cmd_copy = COPY $@
 # send stderr to /dev/null to ignore messages when linking directories.
 cmd_copy = ln -f "$<" "$@" 2>/dev/null || (rm -rf "$@" && cp -af "$<" "$@")
 
-quiet_cmd_symlink = SYMLINK $@
-cmd_symlink = ln -sf "$<" "$@"
-
 quiet_cmd_alink = AR($(TOOLSET)) $@
 cmd_alink = rm -f $@ && $(AR.$(TOOLSET)) crs $@ $(filter %.o,$^)
 
@@ -150,31 +145,6 @@ cmd_alink_thin = rm -f $@ && $(AR.$(TOOLSET)) crsT $@ $(filter %.o,$^)
 quiet_cmd_link = LINK($(TOOLSET)) $@
 cmd_link = $(LINK.$(TOOLSET)) -o $@ $(GYP_LDFLAGS) $(LDFLAGS.$(TOOLSET)) -Wl,--start-group $(LD_INPUTS) $(LIBS) -Wl,--end-group
 
-# Note: this does not handle spaces in paths
-define xargs
-  $(1) $(word 1,$(2))
-$(if $(word 2,$(2)),$(call xargs,$(1),$(wordlist 2,$(words $(2)),$(2))))
-endef
-
-define write-to-file
-  @: >$(1)
-$(call xargs,@printf "%s\n" >>$(1),$(2))
-endef
-
-OBJ_FILE_LIST := ar-file-list
-
-define create_archive
-        rm -f $(1) $(1).$(OBJ_FILE_LIST); mkdir -p `dirname $(1)`
-        $(call write-to-file,$(1).$(OBJ_FILE_LIST),$(filter %.o,$(2)))
-        $(AR.$(TOOLSET)) crs $(1) @$(1).$(OBJ_FILE_LIST)
-endef
-
-define create_thin_archive
-        rm -f $(1) $(OBJ_FILE_LIST); mkdir -p `dirname $(1)`
-        $(call write-to-file,$(1).$(OBJ_FILE_LIST),$(filter %.o,$(2)))
-        $(AR.$(TOOLSET)) crsT $(1) @$(1).$(OBJ_FILE_LIST)
-endef
-
 # We support two kinds of shared objects (.so):
 # 1) shared_library, which is just bundling together many dependent libraries
 # into a link line.
@@ -338,8 +308,8 @@ ifeq ($(strip $(foreach prefix,$(NO_LOAD),\
 endif
 
 quiet_cmd_regen_makefile = ACTION Regenerating $@
-cmd_regen_makefile = cd $(srcdir); /usr/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/mark/.cache/node-gyp/22.18.0" "-Dnode_gyp_dir=/usr/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=/home/mark/.cache/node-gyp/22.18.0/<(target_arch)/node.lib" "-Dmodule_root_dir=/tmp/tmp-5278lDVbQAP2i4IB/node_modules/node-pty" "-Dnode_engine=v8" "--depth=." "-Goutput_dir=." "--generator-output=build" -I/tmp/tmp-5278lDVbQAP2i4IB/node_modules/node-pty/build/config.gypi -I/usr/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/mark/.cache/node-gyp/22.18.0/include/node/common.gypi "--toplevel-dir=." binding.gyp
-Makefile: $(srcdir)/../../../../home/mark/.cache/node-gyp/22.18.0/include/node/common.gypi $(srcdir)/../../../../usr/lib/node_modules/npm/node_modules/node-gyp/addon.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp
+cmd_regen_makefile = cd $(srcdir); /mnt/g/Projects/zoneweaver-api/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/mark/.cache/node-gyp/22.18.0" "-Dnode_gyp_dir=/mnt/g/Projects/zoneweaver-api/node_modules/node-gyp" "-Dnode_lib_file=/home/mark/.cache/node-gyp/22.18.0/<(target_arch)/node.lib" "-Dmodule_root_dir=/mnt/g/Projects/zoneweaver-api/node_modules/node-pty" "-Dnode_engine=v8" "--depth=." "-Goutput_dir=." "--generator-output=build" -I/mnt/g/Projects/zoneweaver-api/node_modules/node-pty/build/config.gypi -I/mnt/g/Projects/zoneweaver-api/node_modules/node-gyp/addon.gypi -I/home/mark/.cache/node-gyp/22.18.0/include/node/common.gypi "--toplevel-dir=." binding.gyp
+Makefile: $(srcdir)/../node-gyp/addon.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../../../home/mark/.cache/node-gyp/22.18.0/include/node/common.gypi
 	$(call do_cmd,regen_makefile)
 
 # "all" is a concatenation of the "all" targets from all the included
diff --git a/node_modules/node-pty/build/binding.Makefile b/node_modules/node-pty/build/binding.Makefile
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/build/config.gypi b/node_modules/node-pty/build/config.gypi
old mode 100644
new mode 100755
index ae25cc4..3bdd04e
--- a/node_modules/node-pty/build/config.gypi
+++ b/node_modules/node-pty/build/config.gypi
@@ -491,12 +491,10 @@
     "v8_use_siphash": 1,
     "want_separate_host_toolset": 0,
     "nodedir": "/home/mark/.cache/node-gyp/22.18.0",
-    "python": "/usr/bin/python3",
     "standalone_static_library": 1,
     "user_agent": "npm/10.9.3 node/v22.18.0 linux x64 workspaces/false",
-    "force": "true",
     "userconfig": "/home/mark/.npmrc",
-    "local_prefix": "/tmp/tmp-5278lDVbQAP2i4IB",
+    "local_prefix": "/mnt/g/Projects/zoneweaver-api",
     "prefix": "/usr",
     "npm_version": "10.9.3",
     "cache": "/home/mark/.npm",
diff --git a/node_modules/node-pty/build/pty.target.mk b/node_modules/node-pty/build/pty.target.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/.editorconfig b/node_modules/node-pty/deps/.editorconfig
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/.drone.yml b/node_modules/node-pty/deps/winpty/.drone.yml
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/.gitattributes b/node_modules/node-pty/deps/winpty/.gitattributes
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/LICENSE b/node_modules/node-pty/deps/winpty/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/Makefile b/node_modules/node-pty/deps/winpty/Makefile
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/README.md b/node_modules/node-pty/deps/winpty/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/RELEASES.md b/node_modules/node-pty/deps/winpty/RELEASES.md
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/VERSION.txt b/node_modules/node-pty/deps/winpty/VERSION.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/configure b/node_modules/node-pty/deps/winpty/configure
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/BufferResizeTests.cc b/node_modules/node-pty/deps/winpty/misc/BufferResizeTests.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ChangeScreenBuffer.cc b/node_modules/node-pty/deps/winpty/misc/ChangeScreenBuffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ClearConsole.cc b/node_modules/node-pty/deps/winpty/misc/ClearConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ConinMode.cc b/node_modules/node-pty/deps/winpty/misc/ConinMode.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ConinMode.ps1 b/node_modules/node-pty/deps/winpty/misc/ConinMode.ps1
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ConoutMode.cc b/node_modules/node-pty/deps/winpty/misc/ConoutMode.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/DebugClient.py b/node_modules/node-pty/deps/winpty/misc/DebugClient.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/DebugServer.py b/node_modules/node-pty/deps/winpty/misc/DebugServer.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/DumpLines.py b/node_modules/node-pty/deps/winpty/misc/DumpLines.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/EnableExtendedFlags.txt b/node_modules/node-pty/deps/winpty/misc/EnableExtendedFlags.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP437-Consolas.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP437-Consolas.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP437-Lucida.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP437-Lucida.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP932.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP932.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP936.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP936.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP949.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP949.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP950.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/CP950.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/MinimumWindowWidths.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/MinimumWindowWidths.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/Results.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/Results.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/Windows10SetFontBugginess.txt b/node_modules/node-pty/deps/winpty/misc/Font-Report-June2016/Windows10SetFontBugginess.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/FontSurvey.cc b/node_modules/node-pty/deps/winpty/misc/FontSurvey.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/FormatChar.h b/node_modules/node-pty/deps/winpty/misc/FormatChar.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/FreezePerfTest.cc b/node_modules/node-pty/deps/winpty/misc/FreezePerfTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/GetCh.cc b/node_modules/node-pty/deps/winpty/misc/GetCh.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/GetConsolePos.cc b/node_modules/node-pty/deps/winpty/misc/GetConsolePos.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/GetFont.cc b/node_modules/node-pty/deps/winpty/misc/GetFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/IdentifyConsoleWindow.ps1 b/node_modules/node-pty/deps/winpty/misc/IdentifyConsoleWindow.ps1
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/IsNewConsole.cc b/node_modules/node-pty/deps/winpty/misc/IsNewConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/MouseInputNotes.txt b/node_modules/node-pty/deps/winpty/misc/MouseInputNotes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/MoveConsoleWindow.cc b/node_modules/node-pty/deps/winpty/misc/MoveConsoleWindow.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Notes.txt b/node_modules/node-pty/deps/winpty/misc/Notes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/OSVersion.cc b/node_modules/node-pty/deps/winpty/misc/OSVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ScreenBufferFreezeInactive.cc b/node_modules/node-pty/deps/winpty/misc/ScreenBufferFreezeInactive.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ScreenBufferTest.cc b/node_modules/node-pty/deps/winpty/misc/ScreenBufferTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ScreenBufferTest2.cc b/node_modules/node-pty/deps/winpty/misc/ScreenBufferTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/SelectAllTest.cc b/node_modules/node-pty/deps/winpty/misc/SelectAllTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/SetBufferSize.cc b/node_modules/node-pty/deps/winpty/misc/SetBufferSize.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/SetCursorPos.cc b/node_modules/node-pty/deps/winpty/misc/SetCursorPos.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/SetFont.cc b/node_modules/node-pty/deps/winpty/misc/SetFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/SetWindowRect.cc b/node_modules/node-pty/deps/winpty/misc/SetWindowRect.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ShowArgv.cc b/node_modules/node-pty/deps/winpty/misc/ShowArgv.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/ShowConsoleInput.cc b/node_modules/node-pty/deps/winpty/misc/ShowConsoleInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Spew.py b/node_modules/node-pty/deps/winpty/misc/Spew.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/TestUtil.cc b/node_modules/node-pty/deps/winpty/misc/TestUtil.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/UnicodeDoubleWidthTest.cc b/node_modules/node-pty/deps/winpty/misc/UnicodeDoubleWidthTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/UnicodeWideTest1.cc b/node_modules/node-pty/deps/winpty/misc/UnicodeWideTest1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/UnicodeWideTest2.cc b/node_modules/node-pty/deps/winpty/misc/UnicodeWideTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/UnixEcho.cc b/node_modules/node-pty/deps/winpty/misc/UnixEcho.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Utf16Echo.cc b/node_modules/node-pty/deps/winpty/misc/Utf16Echo.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/VeryLargeRead.cc b/node_modules/node-pty/deps/winpty/misc/VeryLargeRead.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/VkEscapeTest.cc b/node_modules/node-pty/deps/winpty/misc/VkEscapeTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win10ResizeWhileFrozen.cc b/node_modules/node-pty/deps/winpty/misc/Win10ResizeWhileFrozen.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win10WrapTest1.cc b/node_modules/node-pty/deps/winpty/misc/Win10WrapTest1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win10WrapTest2.cc b/node_modules/node-pty/deps/winpty/misc/Win10WrapTest2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Echo1.cc b/node_modules/node-pty/deps/winpty/misc/Win32Echo1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Echo2.cc b/node_modules/node-pty/deps/winpty/misc/Win32Echo2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Test1.cc b/node_modules/node-pty/deps/winpty/misc/Win32Test1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Test2.cc b/node_modules/node-pty/deps/winpty/misc/Win32Test2.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Test3.cc b/node_modules/node-pty/deps/winpty/misc/Win32Test3.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/Win32Write1.cc b/node_modules/node-pty/deps/winpty/misc/Win32Write1.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/WindowsBugCrashReader.cc b/node_modules/node-pty/deps/winpty/misc/WindowsBugCrashReader.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/WriteConsole.cc b/node_modules/node-pty/deps/winpty/misc/WriteConsole.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/build32.sh b/node_modules/node-pty/deps/winpty/misc/build32.sh
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/build64.sh b/node_modules/node-pty/deps/winpty/misc/build64.sh
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/color-test.sh b/node_modules/node-pty/deps/winpty/misc/color-test.sh
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/font-notes.txt b/node_modules/node-pty/deps/winpty/misc/font-notes.txt
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/misc/winbug-15048.cc b/node_modules/node-pty/deps/winpty/misc/winbug-15048.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/ship/build-pty4j-libpty.bat b/node_modules/node-pty/deps/winpty/ship/build-pty4j-libpty.bat
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/ship/common_ship.py b/node_modules/node-pty/deps/winpty/ship/common_ship.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/ship/make_msvc_package.py b/node_modules/node-pty/deps/winpty/ship/make_msvc_package.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/ship/ship.py b/node_modules/node-pty/deps/winpty/ship/ship.py
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Agent.cc b/node_modules/node-pty/deps/winpty/src/agent/Agent.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Agent.h b/node_modules/node-pty/deps/winpty/src/agent/Agent.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/AgentCreateDesktop.cc b/node_modules/node-pty/deps/winpty/src/agent/AgentCreateDesktop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/AgentCreateDesktop.h b/node_modules/node-pty/deps/winpty/src/agent/AgentCreateDesktop.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleFont.cc b/node_modules/node-pty/deps/winpty/src/agent/ConsoleFont.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleFont.h b/node_modules/node-pty/deps/winpty/src/agent/ConsoleFont.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleInput.cc b/node_modules/node-pty/deps/winpty/src/agent/ConsoleInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleInput.h b/node_modules/node-pty/deps/winpty/src/agent/ConsoleInput.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleInputReencoding.cc b/node_modules/node-pty/deps/winpty/src/agent/ConsoleInputReencoding.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleInputReencoding.h b/node_modules/node-pty/deps/winpty/src/agent/ConsoleInputReencoding.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleLine.cc b/node_modules/node-pty/deps/winpty/src/agent/ConsoleLine.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/ConsoleLine.h b/node_modules/node-pty/deps/winpty/src/agent/ConsoleLine.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Coord.h b/node_modules/node-pty/deps/winpty/src/agent/Coord.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/DebugShowInput.cc b/node_modules/node-pty/deps/winpty/src/agent/DebugShowInput.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/DebugShowInput.h b/node_modules/node-pty/deps/winpty/src/agent/DebugShowInput.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/DefaultInputMap.cc b/node_modules/node-pty/deps/winpty/src/agent/DefaultInputMap.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/DefaultInputMap.h b/node_modules/node-pty/deps/winpty/src/agent/DefaultInputMap.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/DsrSender.h b/node_modules/node-pty/deps/winpty/src/agent/DsrSender.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/EventLoop.cc b/node_modules/node-pty/deps/winpty/src/agent/EventLoop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/EventLoop.h b/node_modules/node-pty/deps/winpty/src/agent/EventLoop.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/InputMap.cc b/node_modules/node-pty/deps/winpty/src/agent/InputMap.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/InputMap.h b/node_modules/node-pty/deps/winpty/src/agent/InputMap.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/LargeConsoleRead.cc b/node_modules/node-pty/deps/winpty/src/agent/LargeConsoleRead.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/LargeConsoleRead.h b/node_modules/node-pty/deps/winpty/src/agent/LargeConsoleRead.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/NamedPipe.cc b/node_modules/node-pty/deps/winpty/src/agent/NamedPipe.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/NamedPipe.h b/node_modules/node-pty/deps/winpty/src/agent/NamedPipe.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Scraper.cc b/node_modules/node-pty/deps/winpty/src/agent/Scraper.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Scraper.h b/node_modules/node-pty/deps/winpty/src/agent/Scraper.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/SimplePool.h b/node_modules/node-pty/deps/winpty/src/agent/SimplePool.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/SmallRect.h b/node_modules/node-pty/deps/winpty/src/agent/SmallRect.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Terminal.cc b/node_modules/node-pty/deps/winpty/src/agent/Terminal.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Terminal.h b/node_modules/node-pty/deps/winpty/src/agent/Terminal.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/UnicodeEncoding.h b/node_modules/node-pty/deps/winpty/src/agent/UnicodeEncoding.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/UnicodeEncodingTest.cc b/node_modules/node-pty/deps/winpty/src/agent/UnicodeEncodingTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Win32Console.cc b/node_modules/node-pty/deps/winpty/src/agent/Win32Console.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Win32Console.h b/node_modules/node-pty/deps/winpty/src/agent/Win32Console.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Win32ConsoleBuffer.cc b/node_modules/node-pty/deps/winpty/src/agent/Win32ConsoleBuffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/Win32ConsoleBuffer.h b/node_modules/node-pty/deps/winpty/src/agent/Win32ConsoleBuffer.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/main.cc b/node_modules/node-pty/deps/winpty/src/agent/main.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/agent/subdir.mk b/node_modules/node-pty/deps/winpty/src/agent/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/configurations.gypi b/node_modules/node-pty/deps/winpty/src/configurations.gypi
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/debugserver/DebugServer.cc b/node_modules/node-pty/deps/winpty/src/debugserver/DebugServer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/debugserver/subdir.mk b/node_modules/node-pty/deps/winpty/src/debugserver/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/include/winpty.h b/node_modules/node-pty/deps/winpty/src/include/winpty.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/include/winpty_constants.h b/node_modules/node-pty/deps/winpty/src/include/winpty_constants.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/AgentLocation.cc b/node_modules/node-pty/deps/winpty/src/libwinpty/AgentLocation.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/AgentLocation.h b/node_modules/node-pty/deps/winpty/src/libwinpty/AgentLocation.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/LibWinptyException.h b/node_modules/node-pty/deps/winpty/src/libwinpty/LibWinptyException.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/WinptyInternal.h b/node_modules/node-pty/deps/winpty/src/libwinpty/WinptyInternal.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/subdir.mk b/node_modules/node-pty/deps/winpty/src/libwinpty/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/libwinpty/winpty.cc b/node_modules/node-pty/deps/winpty/src/libwinpty/winpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/AgentMsg.h b/node_modules/node-pty/deps/winpty/src/shared/AgentMsg.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/BackgroundDesktop.cc b/node_modules/node-pty/deps/winpty/src/shared/BackgroundDesktop.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/BackgroundDesktop.h b/node_modules/node-pty/deps/winpty/src/shared/BackgroundDesktop.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/Buffer.cc b/node_modules/node-pty/deps/winpty/src/shared/Buffer.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/Buffer.h b/node_modules/node-pty/deps/winpty/src/shared/Buffer.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/DebugClient.cc b/node_modules/node-pty/deps/winpty/src/shared/DebugClient.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/DebugClient.h b/node_modules/node-pty/deps/winpty/src/shared/DebugClient.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/GenRandom.cc b/node_modules/node-pty/deps/winpty/src/shared/GenRandom.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/GenRandom.h b/node_modules/node-pty/deps/winpty/src/shared/GenRandom.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/GetCommitHash.bat b/node_modules/node-pty/deps/winpty/src/shared/GetCommitHash.bat
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/Mutex.h b/node_modules/node-pty/deps/winpty/src/shared/Mutex.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/OsModule.h b/node_modules/node-pty/deps/winpty/src/shared/OsModule.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/OwnedHandle.cc b/node_modules/node-pty/deps/winpty/src/shared/OwnedHandle.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/OwnedHandle.h b/node_modules/node-pty/deps/winpty/src/shared/OwnedHandle.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/PrecompiledHeader.h b/node_modules/node-pty/deps/winpty/src/shared/PrecompiledHeader.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/StringBuilder.h b/node_modules/node-pty/deps/winpty/src/shared/StringBuilder.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/StringBuilderTest.cc b/node_modules/node-pty/deps/winpty/src/shared/StringBuilderTest.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/StringUtil.cc b/node_modules/node-pty/deps/winpty/src/shared/StringUtil.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/StringUtil.h b/node_modules/node-pty/deps/winpty/src/shared/StringUtil.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/TimeMeasurement.h b/node_modules/node-pty/deps/winpty/src/shared/TimeMeasurement.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/UnixCtrlChars.h b/node_modules/node-pty/deps/winpty/src/shared/UnixCtrlChars.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/UpdateGenVersion.bat b/node_modules/node-pty/deps/winpty/src/shared/UpdateGenVersion.bat
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WindowsSecurity.cc b/node_modules/node-pty/deps/winpty/src/shared/WindowsSecurity.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WindowsSecurity.h b/node_modules/node-pty/deps/winpty/src/shared/WindowsSecurity.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WindowsVersion.cc b/node_modules/node-pty/deps/winpty/src/shared/WindowsVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WindowsVersion.h b/node_modules/node-pty/deps/winpty/src/shared/WindowsVersion.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyAssert.cc b/node_modules/node-pty/deps/winpty/src/shared/WinptyAssert.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyAssert.h b/node_modules/node-pty/deps/winpty/src/shared/WinptyAssert.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyException.cc b/node_modules/node-pty/deps/winpty/src/shared/WinptyException.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyException.h b/node_modules/node-pty/deps/winpty/src/shared/WinptyException.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyVersion.cc b/node_modules/node-pty/deps/winpty/src/shared/WinptyVersion.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/WinptyVersion.h b/node_modules/node-pty/deps/winpty/src/shared/WinptyVersion.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/shared/winpty_snprintf.h b/node_modules/node-pty/deps/winpty/src/shared/winpty_snprintf.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/subdir.mk b/node_modules/node-pty/deps/winpty/src/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/tests/subdir.mk b/node_modules/node-pty/deps/winpty/src/tests/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/tests/trivial_test.cc b/node_modules/node-pty/deps/winpty/src/tests/trivial_test.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/InputHandler.cc b/node_modules/node-pty/deps/winpty/src/unix-adapter/InputHandler.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/InputHandler.h b/node_modules/node-pty/deps/winpty/src/unix-adapter/InputHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/OutputHandler.cc b/node_modules/node-pty/deps/winpty/src/unix-adapter/OutputHandler.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/OutputHandler.h b/node_modules/node-pty/deps/winpty/src/unix-adapter/OutputHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/Util.cc b/node_modules/node-pty/deps/winpty/src/unix-adapter/Util.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/Util.h b/node_modules/node-pty/deps/winpty/src/unix-adapter/Util.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/WakeupFd.cc b/node_modules/node-pty/deps/winpty/src/unix-adapter/WakeupFd.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/WakeupFd.h b/node_modules/node-pty/deps/winpty/src/unix-adapter/WakeupFd.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/main.cc b/node_modules/node-pty/deps/winpty/src/unix-adapter/main.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/unix-adapter/subdir.mk b/node_modules/node-pty/deps/winpty/src/unix-adapter/subdir.mk
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/src/winpty.gyp b/node_modules/node-pty/deps/winpty/src/winpty.gyp
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/deps/winpty/vcbuild.bat b/node_modules/node-pty/deps/winpty/vcbuild.bat
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/conpty_console_list_agent.js b/node_modules/node-pty/lib/conpty_console_list_agent.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/conpty_console_list_agent.js.map b/node_modules/node-pty/lib/conpty_console_list_agent.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/eventEmitter2.js b/node_modules/node-pty/lib/eventEmitter2.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/eventEmitter2.js.map b/node_modules/node-pty/lib/eventEmitter2.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/eventEmitter2.test.js b/node_modules/node-pty/lib/eventEmitter2.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/eventEmitter2.test.js.map b/node_modules/node-pty/lib/eventEmitter2.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/index.js b/node_modules/node-pty/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/index.js.map b/node_modules/node-pty/lib/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/interfaces.js b/node_modules/node-pty/lib/interfaces.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/interfaces.js.map b/node_modules/node-pty/lib/interfaces.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/shared/conout.js b/node_modules/node-pty/lib/shared/conout.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/shared/conout.js.map b/node_modules/node-pty/lib/shared/conout.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/terminal.js b/node_modules/node-pty/lib/terminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/terminal.js.map b/node_modules/node-pty/lib/terminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/terminal.test.js b/node_modules/node-pty/lib/terminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/terminal.test.js.map b/node_modules/node-pty/lib/terminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/testUtils.test.js b/node_modules/node-pty/lib/testUtils.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/testUtils.test.js.map b/node_modules/node-pty/lib/testUtils.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/types.js b/node_modules/node-pty/lib/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/types.js.map b/node_modules/node-pty/lib/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/unixTerminal.js b/node_modules/node-pty/lib/unixTerminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/unixTerminal.js.map b/node_modules/node-pty/lib/unixTerminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/unixTerminal.test.js b/node_modules/node-pty/lib/unixTerminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/unixTerminal.test.js.map b/node_modules/node-pty/lib/unixTerminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/utils.js b/node_modules/node-pty/lib/utils.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/utils.js.map b/node_modules/node-pty/lib/utils.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsConoutConnection.js b/node_modules/node-pty/lib/windowsConoutConnection.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsConoutConnection.js.map b/node_modules/node-pty/lib/windowsConoutConnection.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsPtyAgent.js b/node_modules/node-pty/lib/windowsPtyAgent.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsPtyAgent.js.map b/node_modules/node-pty/lib/windowsPtyAgent.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsPtyAgent.test.js b/node_modules/node-pty/lib/windowsPtyAgent.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsPtyAgent.test.js.map b/node_modules/node-pty/lib/windowsPtyAgent.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsTerminal.js b/node_modules/node-pty/lib/windowsTerminal.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsTerminal.js.map b/node_modules/node-pty/lib/windowsTerminal.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsTerminal.test.js b/node_modules/node-pty/lib/windowsTerminal.test.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/windowsTerminal.test.js.map b/node_modules/node-pty/lib/windowsTerminal.test.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/worker/conoutSocketWorker.js b/node_modules/node-pty/lib/worker/conoutSocketWorker.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/lib/worker/conoutSocketWorker.js.map b/node_modules/node-pty/lib/worker/conoutSocketWorker.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/scripts/post-install.js b/node_modules/node-pty/scripts/post-install.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/scripts/publish.js b/node_modules/node-pty/scripts/publish.js
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/conpty_console_list_agent.ts b/node_modules/node-pty/src/conpty_console_list_agent.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/eventEmitter2.test.ts b/node_modules/node-pty/src/eventEmitter2.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/eventEmitter2.ts b/node_modules/node-pty/src/eventEmitter2.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/index.ts b/node_modules/node-pty/src/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/interfaces.ts b/node_modules/node-pty/src/interfaces.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/native.d.ts b/node_modules/node-pty/src/native.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/shared/conout.ts b/node_modules/node-pty/src/shared/conout.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/terminal.test.ts b/node_modules/node-pty/src/terminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/terminal.ts b/node_modules/node-pty/src/terminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/testUtils.test.ts b/node_modules/node-pty/src/testUtils.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/tsconfig.json b/node_modules/node-pty/src/tsconfig.json
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/types.ts b/node_modules/node-pty/src/types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/unix/pty.cc b/node_modules/node-pty/src/unix/pty.cc
old mode 100644
new mode 100755
index cb5649e..0fd56e2
--- a/node_modules/node-pty/src/unix/pty.cc
+++ b/node_modules/node-pty/src/unix/pty.cc
@@ -29,6 +29,7 @@
 #include <sys/wait.h>
 #include <fcntl.h>
 #include <signal.h>
+#include <cstddef>          /* for size_t */
 
 /* forkpty */
 /* http://www.gnu.org/software/gnulib/manual/html_node/forkpty.html */
@@ -38,6 +39,12 @@
 #include <util.h>
 #elif defined(__FreeBSD__)
 #include <libutil.h>
+#elif defined(__sun) || defined(__SunOS)
+#include <termios.h>
+#include <sys/termios.h>
+#include <stropts.h>        /* for I_PUSH - from original Microsoft PR */
+#include <sys/stropts.h>    /* additional streams support */
+#include <libutil.h>        /* for forkpty/openpty on Solaris */
 #endif
 
 /* Some platforms name VWERASE and VDISCARD differently */
diff --git a/node_modules/node-pty/src/unix/spawn-helper.cc b/node_modules/node-pty/src/unix/spawn-helper.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/unixTerminal.test.ts b/node_modules/node-pty/src/unixTerminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/unixTerminal.ts b/node_modules/node-pty/src/unixTerminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/utils.ts b/node_modules/node-pty/src/utils.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/win/conpty.cc b/node_modules/node-pty/src/win/conpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/win/conpty_console_list.cc b/node_modules/node-pty/src/win/conpty_console_list.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/win/path_util.cc b/node_modules/node-pty/src/win/path_util.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/win/path_util.h b/node_modules/node-pty/src/win/path_util.h
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/win/winpty.cc b/node_modules/node-pty/src/win/winpty.cc
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/windowsConoutConnection.ts b/node_modules/node-pty/src/windowsConoutConnection.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/windowsPtyAgent.test.ts b/node_modules/node-pty/src/windowsPtyAgent.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/windowsPtyAgent.ts b/node_modules/node-pty/src/windowsPtyAgent.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/windowsTerminal.test.ts b/node_modules/node-pty/src/windowsTerminal.test.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/windowsTerminal.ts b/node_modules/node-pty/src/windowsTerminal.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/src/worker/conoutSocketWorker.ts b/node_modules/node-pty/src/worker/conoutSocketWorker.ts
old mode 100644
new mode 100755
diff --git a/node_modules/node-pty/typings/node-pty.d.ts b/node_modules/node-pty/typings/node-pty.d.ts
old mode 100644
new mode 100755
